packages/components/src/button/index.js

```diff
@@ -33,14 +33,16 @@ export function Button( props, ref ) {
 		'is-large': isLarge,
 		'is-small': isSmall,
 		'is-tertiary': isTertiary,
-		'is-toggled': isToggled,
+		'is-pressed': isPressed,
 		'is-busy': isBusy,
 		'is-link': isLink,
 		'is-destructive': isDestructive,
 	} );
 
 	const tag = href !== undefined && ! disabled ? 'a' : 'button';
-	const tagProps = tag === 'a' ? { href, target } : { type: 'button', disabled };
+	const tagProps = tag === 'a' ?
+		{ href, target } :
+		{ type: 'button', disabled, 'aria-pressed': isPressed };
```

Comment:
Do we have default styles for the isPressed variation of the Button

Reply:
As commented: no, we don’t, we should do it as the follow up.

Reply:
Looks like `isBoolean` is the convention adopted by Gutenberg, right? Should `disabled` be `isDisabled` as well? I see that both are used in the project (for example, `ToolbarButton` currently accepts `isDisabled` and passes it to the `disabled` prop on `Button`).

Reply:
@aduth - do you have recommendations here? I don't think we made it the rule. I'm happy to open a follow-up to update if necessary. `disabled` will always work because it's HTML attribute ... but it would be nice to follow conventions.

I also want to have `isFocusable` prop so we don't have to use `<Button aria-disabled={ true } />` as a workaround for `<Button isDisabled isFocusable />`.

Reply:
I used to prefer the un-prefixed `disabled` over an `isDisabled`, given that it aligned better with expectations around how attributes apply to HTML elements, but over time I've come to regard this as an alignment that we _shouldn't want to_ try to reinforce. In the worst case, it can mislead a developer to think that their `disabled` prop would be guaranteed to apply a `disabled` HTML attribute somewhere, which isn't always the case (we should embrace that a component can be an abstraction not directly tied to some specific DOM implementation). Conversely, we lose any benefit that we can gain from "signalling" an expected prop value; a prop named `foo` could be interpreted to be any value, but an equivalent `isFoo` is more clearly expecting a boolean.

My recommendation would be to prefer to name new boolean props using the `isFoo` convention.